packages/openapi3/test/primitive-types.test.ts

```diff
@@ -48,6 +48,19 @@ describe("openapi3: primitives", () => {
     }
   });
 
+  it("produce type: integer, format: double-int for safeint when safeint-strategy is double-int", async () => {
```

Comment:
Need to add a test to ensure that the value is base64 when no value is specified (do we have this already)?  Also, should ensure we get base64 when this is officially set.

Reply:
Also, we should have a test in which an `@encode` is added, which I think should win out over this setting

Reply:
you mean int64? we do have the test above that deal with the default

Reply:
I can add an explicit with the option as well